app/src/main/java/com/example/pix/chat/activities/FriendActivity.java

```diff
@@ -17,17 +17,20 @@
 
 public class FriendActivity extends AppCompatActivity {
 
+    public static final String FRIEND_FRAGMENT_TAG = "friend";
+
     @RequiresApi(api = Build.VERSION_CODES.M)
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_friend);
 
         // Check if we took a picture from HomeFragment's ComposeFragment
+        // Set a tag for this Fragment so we can retrieve it if we go to this friend's ProfileFragment
```

Comment:
I need to retrieve this FriendFragment in the friend's ProfileFragment so that I may hide the ProfileFragment and show the FriendFragment.